### ISSUE TITLE
Consider moving the repository to `nuxt-modules/gtag`

### ISSUE THREAD

**Issue by DamianGlowala** [johannschopplich/nuxt-gtag#60]
### Describe the feature

It might be a good time to consider moving this repository to `nuxt-modules/gtag`. Would you be open to this idea?

https://nuxt.com/docs/guide/going-further/modules#joining-nuxt-modules-and-nuxtjs

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Can you think of other implementations of this feature?

### Final checks

- [X] Check existing [issues](https://github.com/johannschopplich/nuxt-gtag/issues).

**Comment by johannschopplich** [comment]
Yeah, I'd gladly move this repo to Nuxt Modules! I'm maintaining [Nuxt Plausible](https://github.com/nuxt-modules/plausible) already, so it would be a good fit to move this repo as well.

How do we continue. Should I [open an issue in nuxt/modules](https://github.com/nuxt/modules/issues/new)?

**Comment by DamianGlowala** [comment]
Yes please! Creating an issue would be a good starting point :) I'll try to move this forward if it gets stuck for too long without any action being taken.
